core/client/src/main/java/alluxio/hadoop/HadoopUtils.java

```diff
@@ -148,11 +148,11 @@ public static String toStringHadoopInputSplit(InputSplit is) {
    * @param conf Hadoop configuration
    */
   public static void addS3Credentials(Configuration conf) {
```

Comment:
not really related to this PR, but should we have an `addGCSCredentials` method to propagate the GCS settings to Hadoop?

@ifcharming